tensorboard/plugins/pr_curve/summary_test.py

```diff
@@ -75,27 +75,29 @@ def test1Class(self):
 
     tensor_nd_array = tf.make_ndarray(tensor_event.tensor_proto)
 
-    # The tensor shape must be correct. The first dimension is the
-    # type of value (see documentation for the op). The 2nd dimension
-    # is the number of classes. The last dimension is the number of
-    # thresholds.
-    correct_shape = [6, 1, 10]
+    # The tensor shape must be correct. The first dimension is the value to
+    # compute (see documentation for the op or the comments below). The second
+    # dimension is the number of thresholds.
+    correct_shape = [6, 10]
     self.assertListEqual(correct_shape, list(tensor_nd_array.shape))
 
+    print(`tensor_nd_array.tolist()`)
```

Comment:
Do not use backticks; write `repr` instead. Backticks are generally discouraged (citation needed) and removed in Python 3 because they were generally discouraged (citation needed).

But really just remove this statement.